nss-day-cohort-51/holiday-road-grand-canyon-gliders
# Bw event park name

# Description

Changed the styling of the Event Modal


## Type of change

Please delete options that are not relevant.


# Testing Instructions

To test click the events button under saved trips to see the park name is displayed if an event is present.



# Checklist:
